extensions/amp-access/0.1/amp-access.js

```diff
@@ -16,6 +16,7 @@
 
 import {actionServiceFor} from '../../../src/action';
 import {assertHttpsUrl} from '../../../src/url';
+import {installCidService} from '../../../src/service/cid-impl.js';
```

Comment:
My assumption for now that no - amp-analytics is not necessary. I'd be fine to require it now, but I was planning to reduce dependencies here.